packages/ComponentLibrary/src/components/NotificationsButton/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import {
+  Badge,
+  IconButton,
+  List,
+  Menu,
+  MenuItem,
+  Tooltip,
+} from '@mui/material';
+import { NOTIFICATIONS } from './mock';
+import { NotificationsOutlined } from '@mui/icons-material';
+import { styles, sx } from './styles';
+import { NotificationButtonProps } from './types';
+
+const NotificationButton: React.FC<NotificationButtonProps> = ({
+  notifications,
+  onClick,
+}) => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const notificationCount = NOTIFICATIONS.length;
```

Comment:
Shouldn't you take this value from the prop and not insert it statically from the component?